mephisto/abstractions/crowd_provider.py

```diff
@@ -122,6 +122,10 @@ def setup_resources_for_task_run(
         """
         raise NotImplementedError()
 
+    @abstractmethod
+    def print_banner_on_start(self) -> str:
+        return ""
```

Comment:
Nit: if this isn't printing itself, it shouldn't have a name in the format `print_*`. `get_start_banner()` or similar could work.

That being said, I think it actually makes more sense for this method to do the printing, as it can also be responsible for confirm steps, like:
```
You are about to launch 50,000 tasks, for an estimated total of $1,000,000, are you sure you want to do that...?
```

Unfortunately, if we're adding confirm steps, it's also possible that we want to skip the confirmations entirely, so that's something to account for.